docs/wizards.md

```diff
@@ -0,0 +1,195 @@
+# Wizards
+
+To simplify the process of creating multi-step processes with no intermediate persistence, a number of reusable model and controller modules have been implemented. They require minimal configuration and make using wizards a lot easier.
+
+- [Wizard](#wizard)
+- [Wizard::PathHistory](#wizardpathhistory)
+- [ClearWizardCache](#clearwizardcache)
+- [Things to note](#thingstonote)

```

Comment:
Not sure why this isn't linking out, might be just the PR view?

Reply:
Fixed now, I should have used dashes to separate the words.